dockyard/common/container/container.py

```diff
@@ -0,0 +1,162 @@
+from dockyard.common import utils, link
```

Comment:
We have to add documentation for each module, method and class. We have to make some standard for documentation also. Some research might be done on it.